Updated Readme about features and commands (#59)
Platform-OS/pos-module-core: README.md

# platformOS Core Module

The goal of this module is to extend the platformOS module system's possibilities.

## Features

- **Hook system** to apply **SOLID's Open/Closed Principle**, allowing you to modify the business logic of other modules in the application folder without changing the source code of existing modules. Use the **Headscripts Hook** to register and aggregate head scripts (CSS, JS, metadata) from other modules and themes, ensuring a cohesive layout and style across your application.

- **Command Generators**: Generate commands using the provided generators, streamlining your development process with structured code scaffolding.

- **Event System**: Use events to record actions and trigger follow-up actions within your application, facilitating clean code separation and asynchronous processing.

- **Global Variable Storage** to set and retrieve variables in the global scope, providing a consistent way to manage data across your application.

> - **Module Registry & Dependency Management**: Register your module and theme with the **module registry** using `hook_module_info`. This info file allows you to define your module's name, version, type (module or theme), and dependencies. The module registry takes care of **dependency management** and **outdated versions**.

Reviewer: ```suggestion
- **Module Registry & Dependency Management**: Register yourtheme with the **module registry** using `hook_module_info`. This info file allows you to define your theme's name, version, type (theme), and dependencies. The module registry handles **dependency management** and **outdated versions**.
```
